sdcm/sct_config.py

```diff
@@ -1702,6 +1703,16 @@ def _validate_seeds_number(self):
         assert not num_of_db_nodes or seeds_num <= num_of_db_nodes, \
             f"Seeds number ({seeds_num}) should be not more then nodes number ({num_of_db_nodes})"
 
+    def _validate_nemesis_can_run_on_non_seed(self) -> None:
```

Comment:
We need to skip it when `nemesis_class_name=NoOpMonkey`

Reply:
fixed